src/topp/MzTabExporter.cpp

```diff
@@ -211,10 +211,17 @@ namespace OpenMS
         FeatureXMLFile f;
         f.load(in_feature, feature_map);
 
-        vector<ProteinIdentification> prot_ids = feature_map.getProteinIdentifications();
-        ProteinIdentification::SearchParameters sp = prot_ids[0].getSearchParameters();
-        vector<String> var_mods = sp.variable_modifications;
-        vector<String> fixed_mods = sp.fixed_modifications;
+        vector<ProteinIdentification> prot_ids = feature_map.getProteinIdentifications();        
+        vector<String> var_mods, fixed_mods;
+        MzTabString db, db_version;
+        if (!prot_ids.empty())
+        {
+          ProteinIdentification::SearchParameters sp = prot_ids[0].getSearchParameters();
+          var_mods = sp.variable_modifications;
+          fixed_mods = sp.fixed_modifications;
+          db = sp.db.empty() ? MzTabString() : MzTabString(sp.db);
+          db_version = sp.db_version.empty() ? MzTabString() : MzTabString(sp.db_version);
+        }
```

Comment:
what happens when prot_ids is empty? wouldnt it make sense to set db, db_version to some default value (or is that implicitely done upon construction) ?